include/sys/util.h

```diff
@@ -280,7 +280,11 @@ uint8_t u8_to_dec(char *buf, uint8_t buflen, uint8_t value);
 #endif /* !_ASMLANGUAGE */
 
 /** @brief Number of bytes in @p x kibibytes */
+#ifndef _LINKER
```

Comment:
Why this ifdef, why not cast always to size_t?

Reply:
Linker scripts are also using this macro and they would fail with the cast.

Reply:
Could this be documented here, it is not obvious why there is this check.
I have noticed that at least for me it is easier to read `#ifdef _LINKER` instead of the negated one, so I suggest the cases to be switched.

Reply:
Done